js/src/wasm/WasmFrame.h

```diff
@@ -225,6 +225,8 @@ class DebugFrame {
       uint32_t hasSpilledRefRegisterResult : MaxRegisterResults;
     };
     uint32_t allFlags;
+
+    constexpr Flags(uint32_t allFlags) : allFlags(allFlags) {}
```

Comment:
If we just put `allFlags` first in the union, we can initialize `_flags{expr}` with a `uint32_t expr`.